1.8/usage/cli/command-reference.md

```diff
@@ -118,6 +118,57 @@ menu_order: 5
         <subcommand>
             The subcommand name.
     
+## dcos job
+
+Description:
+    Deploy and manage jobs in DC/OS.
+
+    Usage:
+        dcos job [--help]
+        dcos job [--version]
+        dcos job [--config-schema]
+        dcos job [--info]
+        dcos job add <job-file>
+        dcos job remove <job-id> [--stopCurrentJobRuns]
+        dcos job show <job-id>
+        dcos job update <job-file>
+        dcos job kill <job-id> [run-id][--all]
+        dcos job run <job-id>
+        dcos job list [--json]
+        dcos job schedule add <job-id> <schedule-file>
+        dcos job schedule show <job-id> [--json]
+        dcos job schedule remove <job-id> <schedule-id>
+        dcos job schedule update <job-id> <schedule-file>
+        dcos job show runs <job-id> [--run-id <run-id>][--q]
+        dcos job history <job-id> [--json][--show-failures]
+
+Commands:
+    job add
+        Add a job.
+    job remove
+        Remove a job.
+    job show
+        Show the job.
+    job kill
+        Show the job.
+    job run
+        Run a job now.
+    job list
+        Show all job definitions.
+    job schedule
+        Provides ability to create, update and remove schedules for a job.
+    job history
+        Provides a job run history.
+
+Options:
+    -h, --help
+        Print usage.
+    --version
+        Print version information.
+
+Positional Arguments:
+    <job-id>
+        The job ID.
 
 ## dcos marathon
 
```

Comment:
This just changed to be more complete... sorry about that:  here it is:

```
Description:
    Deploy and manage jobs in DC/OS.

    Usage:
        dcos job [--help]
        dcos job [--version]
        dcos job [--config-schema]
        dcos job [--info]
        dcos job add <job-file>
        dcos job remove <job-id> [--stopCurrentJobRuns]
        dcos job show <job-id>
        dcos job update <job-file>
        dcos job kill <job-id> [run-id][--all]
        dcos job run <job-id>
        dcos job list [--json]
        dcos job schedule add <job-id> <schedule-file>
        dcos job schedule show <job-id> [--json]
        dcos job schedule remove <job-id> <schedule-id>
        dcos job schedule update <job-id> <schedule-file>
        dcos job show runs <job-id> [--run-id <run-id>][--json][--q]
        dcos job history <job-id> [--json][--show-failures]

Commands:
    job add
        Add a job.
    job remove
        Remove a job.
    job show
        Show the job.
    job update
        Update the job.
    job kill
        Show the job.
    job run
        Run a job now.
    job list
        Show all job definitions.
    job schedule add
        Adds a schedule to a job.
    job schedule show
        Shows the schedules of a job.
    job schedule remove
        Removes a schedule from a job.
    job schedule update
        Updates a schedule on a job.
    job show runs
        Shows the successful and failure runs of a job.
    job history
        Provides a job run history.

Options:
    -h, --help
        Print usage.
    --version
        Print version information.
    --config-schema
        Show the configuration schema for the Metronome subcommand.
    --info
        Print a short description of this subcommand.
    --stopCurrentJobRuns
        Indicates on a job removal that all current running jobs should be killed.
    --all
        Instead of specifying a run_id to kill, indicates all runs should be killed.
    --json
        Print JSON-formatted list instead of a table.
    --q
        Indicates a quiet mode which results in just an array of run ids.
    --show-failures
        Indicates to show the failure table and statistics for history.

Positional Arguments:
    <job-id>
        The job ID.
    <job-file>
        A JSON formatted file of a job.
    <run-id>
        The run ID of a job run.
    <schedule-file>
        A JSON formatted file of a job schedule.
    <schedule-id>
        The schedule ID.
```